pubtools/_quay/push_docker.py

```diff
@@ -546,7 +552,12 @@ def run(self):
         container_pusher = ContainerImagePusher(docker_push_items, self.target_settings)
         container_pusher.push_container_images()
 
```

Comment:
Can we iterate over the push items here and add missing digests

Reply:
What? Add them where? And why?

Reply:
After push_container_images we iterate over push items and based on media type of push item, other media types will be fetched form quay.io - or rather digests for manifests of those media types. With that we can push v2ch1 or v2ch2 only and when it's about to sign the manifests, we can sign both sch1 and sch2 without need to take care of what is already signed or what isn't.

Reply:
I think you want me to solve the issue of signing V2S2 manifests which were previously only V2S1. I think that code should be a part of #101. Signing V2S2 images after a push constitutes a special case and should not be baked into a normal workflow. I suggest you create a separate function to sign these digests and that this function will be easy to remove after migration.

Reply:
Ok, then lets merge this one and I'll implement signing in #101